docs/fundamentals/executor/executor-in-flow.md

```diff
@@ -0,0 +1,462 @@
+    def bar(self, **kwargs):
+        pass
+
+
+exec_yaml = """ 
+jtype: MyExecutor
+with:
+  parameter_1: static_parameter_1
+  parameter_2: static_parameter_2
+metas:
+  name: MyExecutor
+  description: "MyExecutor does a thing to the stuff in your Documents"
+  workspace: workspace
+requests:
+  /index: default_fn
+  /search: default_fn
+"""
+
+flow1 = Flow().add(uses=exec_yaml)
+
+print(f'\nStarting Flow with default Executor parameters')
+with flow1:
+    pass
+
+flow2 = Flow().add(uses=exec_yaml,
+                   uses_with={'parameter_1': 'overriden_parameter_1', 'parameter_2': 'overriden_parameter_2'},
+                   uses_metas={'name': 'Dynamic Name', 'workspace': 'overriden_worskpace'},
+                   uses_requests={'/index': 'foo', '/search': 'bar'})
+
+print(f'\nStarting Flow with overriden Executor parameters')
+with flow2:
+    pass
+```
+
+```console
+Starting Flow with default Executor parameters
+⠋ 0/2 waiting executor0 gateway to be ready... 
+parameter_1: static_parameter_1
+parameter_2: static_parameter_2
+metas: namespace(description='MyExecutor does a thing to the stuff in your Documents', name='MyExecutor', py_modules='', workspace='workspace')
+requests: {'/default': <function MyExecutor.bar at 0x7fc3163f0710>, '/index': <function MyExecutor.default_fn at 0x7fc3163cc050>, '/search': <function MyExecutor.default_fn at 0x7fc3163cc050>}
+           Flow@21607[I]:🎉 Flow is ready to use!
+	🔗 Protocol: 		GRPC
+	🏠 Local access:	0.0.0.0:53268
+	🔒 Private network:	192.168.1.187:53268
+	🌐 Public address:	212.231.186.65:53268
+
+Starting Flow with overriden Executor parameters
+⠋ 0/2 waiting executor0 gateway to be ready... 
+parameter_1: overriden_parameter_1
+parameter_2: overriden_parameter_2
+metas: namespace(description='MyExecutor does a thing to the stuff in your Documents', name='Dynamic Name', py_modules='', workspace='overriden_worskpace')
+requests: {'/index': <function MyExecutor.foo at 0x7fc3163ddb90>, '/search': <function MyExecutor.bar at 0x7fc3163f0710>}
+           Flow@21607[I]:🎉 Flow is ready to use!
+	🔗 Protocol: 		GRPC
+	🏠 Local access:	0.0.0.0:58267
+	🔒 Private network:	192.168.1.187:58267
+	🌐 Public address:	212.231.186.65:58267
+```
+
+## Pass/change request parameters
+
+An important feature of `Executors` beyond its capacity of `processing` Documents is its capacity to receive `parameters` and to return `dictionaries` from their `methods`. 
+
+In this example, `MyExec` receives the parameters `{'top_k': 10}` from the client and exposes and endpoint returning internal information in the form of dict.
+
+```python
+from jina import requests, Executor, Flow
+
+
+class MyExec(Executor):
+
+    def __init__(self, parameter=20, **kwargs):
+        super().__init__(**kwargs)
+        self.parameter = parameter
+
+    @requests(on='/status')
+    def status(self, **kwargs):
+        return {'internal_parameter': self.parameter}
+
+    @requests(on='/index')
+    def search(self, parameters, **kwargs):
```

Comment:
Would it be more illustrative to save the parameters, and then return them on status, instead of returning something else?